lib/Backends/OpenCL/OpenCLDeviceManager.cpp

```diff
@@ -293,7 +303,7 @@ void OpenCLDeviceManager::evictNetworkImpl(std::string functionName,
   if (evictCB) {
     evictCB(functionName, std::move(err));
   } else {
-    llvm::errs() << llvm::toString(std::move(err));
+    LOG(ERROR) << llvm::toString(std::move(err));
```

Comment:
Might just be me but std err and LOG(ERROR) feel like different severities. Either way this is kind of a weird log line: either it's invalid not to have an `evictCB` in which case this should probably be a DCHECK, or it's valid in which case we probably don't want to spam the log with these.

Reply:
Yeah good point, my preference would be to have it be invalid to not have an evictCB

Reply:
Actually we should just do this for most/all CBs in the runtime